### PR TITLE
RNBS-1053: SqlUpdate Unparsing

### PR DESCRIPTION
**Pull request by nosheenSheikh02**
https://datametica.atlassian.net/browse/RNBS-1053

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/sql/SqlUpdate.java [hunk 1]
```diff
@@ -160,12 +166,104 @@ public void setSourceSelect(SqlSelect sourceSelect) {
...
+    unparseSet(writer, opLeft, opRight);
+    if (!sources.isEmpty()) {
+      unparseUpdateSources(sources, writer, opLeft, opRight);
+    }
+    unparseUpdateCondition(writer, opLeft, opRight);
+
+    writer.endList(frame);
+  }
+
+  private SqlIdentifier getFromAlias() {
```
Comment (deepakkadam-dm) [inline core/src/main/java/org/apache/calcite/sql/SqlUpdate.java:196]: Can the method name be changed to something more elaborate like "getAliasForFromClause"

File: core/src/main/java/org/apache/calcite/sql/SqlUpdate.java [hunk 2]
```diff
@@ -160,12 +166,104 @@ public void setSourceSelect(SqlSelect sourceSelect) {
...
+
+  private void setTargetAndSources(SqlBasicCall node, List<SqlNode> sources) {
+    if (isTargetTable(node)) {
+      setTargetTable(node);
+    } else {
+      sources.add(node);
+    }
+  }
+
+  private boolean isTargetTable(SqlNode node) {
```
Comment (deepakkadam-dm) [inline core/src/main/java/org/apache/calcite/sql/SqlUpdate.java:252]: Can you document about the 3 cases handled as part of this method.

File: core/src/main/java/org/apache/calcite/sql/SqlUpdate.java (hunk 1 above)
Comment (nosheenSheikh02) [reply core/src/main/java/org/apache/calcite/sql/SqlUpdate.java]: Done

File: core/src/main/java/org/apache/calcite/sql/SqlUpdate.java (hunk 2 above)
Comment (nosheenSheikh02) [reply core/src/main/java/org/apache/calcite/sql/SqlUpdate.java:249]: Done

**Review by deepakkadam-dm — APPROVED** [review]
Looks good

**Pull request merged**